### PR TITLE
Show the destination in the domains navigation item for name servers and DNS

### PR DESCRIPTION
**Pull request by hambai**
#### Changes proposed in this Pull Request

* Show the destination of a domain - does it point to WordPress.com site or is external.

It looks like this:

Screenshot 2020-04-13 at 22 27 21
Screenshot 2020-04-13 at 22 27 52
Screenshot 2020-04-13 at 22 26 05

#### Testing instructions

* Open the new domain section and check the "Change your name servers and DNS records" menu for domains with/without our name servers

### CODE REVIEW

File: client/my-sites/domains/domain-management/edit/navigation/enhanced.jsx [hunk 1]
```diff
@@ -101,13 +101,33 @@ class DomainManagementNavigationEnhanced extends React.Component {
 			return null;
 		}
 
-		// NOTE: remember to add translate to the description string once you start working on it
+		const { wpcom_url: wpcomUrl } = selectedSite;
```
Comment (eltongo) [inline client/my-sites/domains/domain-management/edit/navigation/enhanced.jsx:104]: I'm not sure `wpcom_url` is always set. In my case this is leading to weird results. Check the main comment

Comment (hambai) [reply client/my-sites/domains/domain-management/edit/navigation/enhanced.jsx]: Good catch - my test Atomic site was not using custom domain and I totally missed that one

**Comment by fditrapani** [comment]
Thanks for putting this together. This works well for simple sites but I'm seeing this on an atomic site for a primary domain:

image

It should be showing the wpcomstaging url instead.

Another thing I noticed for a simple site was that after changing my name servers to point back to WordPress.com, it continued to show the external message (even with a couple refreshes):

image

**Comment by hambai** [comment]
> This works well for simple sites but I'm seeing this on an atomic site for a primary domain:

Can you confirm that's still what you see for the atomic site? My last commit addressed that issue and it seems to work properly.

> Another thing I noticed for a simple site was that after changing my name servers to point back to WordPress.com, it continued to show the external message (even with a couple refreshes):

This one actually deserves a separate discussion, but it's related to how DNS records are propagated and how we check if a domain points to us or not - I'll create a card and we can discuss end tackle this

**Comment by fditrapani** [comment]
Yeah, I'm still seeing it like this since you merged:

image


> This one actually deserves a separate discussion, but it's related to how DNS records are propagated and how we check if a domain points to us or not - I'll create a card and we can discuss end tackle this

Sounds good. I wonder if we can tie this to the nameserver toggle being on for WordPress.com?

**Comment by akirk** [comment]
@fditrapani Could we improve the wording for

> Destination: external to WordPress.com

To me it sounds like "External → WordPress.com" but I think we are trying to say

> Destination: Not on WordPress.com

Thanks!

**Comment by fditrapani** [comment]
Thanks for bringing this up @akirk! 

> I think we are trying to say destination: Not on WordPress.com
Yes, that's right.  

Here are some other options. Let me know if any of these work:
- Destination: away from WordPress.com
- Destination: outside WordPress.com
- Destination: external service
- Destination: 3rd party service

**Pull request merged**